packages/development-tools/src/build.ts

```diff
@@ -66,11 +65,6 @@ const getStyleLoaders = (cssOptions: any, preProcessor?: string) => {
 }
 
 export function baseJBrowsePluginWebpackConfig(
-  myWebpack: {
-    optimize: {
-      LimitChunkCountPlugin: typeof webpack.optimize.LimitChunkCountPlugin
-    }
-  },
```

Comment:
@rbuels the `myWebpack` param was unused. I removed it, but that changes the exported function's signature. I can put it back if it's supposed to be used or if it needs to stay consistent.